packs.json

```diff
@@ -358,5 +358,14 @@
         "name": "City of Secrets",
         "position": 2,
         "size": 20
+    },
+    {
+        "cgdb_id": 52,
+        "code": "FotS",
+        "cycle_code": "storm",
+        "date_release": "2018-12-27",
```

Comment:
`date_release` must be `NULL`.

Reply:
handled.

Reply:
Whoops, careless copy-paste strikes again. Thanks for fixing!